src/components/pages/complete-password-reset.js

```diff
@@ -130,13 +130,24 @@ class CompletePasswordReset extends React.Component {
       </div>
     );
 
+    const completionFailureNotice = (
+      <div className="CompletePasswordResetPage-message">
```

Comment:
Also here the naming is a bit confusing, Password reset vs changing.

Reply:
Now I'm confused, cause here the naming is already "reset".

Reply:
But the next line says change :)